packages/playout-gateway/src/coreHandler.ts

```diff
@@ -653,16 +646,19 @@ export class CoreTSRDeviceHandler {
 			)
 	}
 
-	async dispose(): Promise<void> {
+	async dispose(expected: boolean): Promise<void> {
```

Comment:
Lindved's Example show how i'd suggest the input of dispose to be, from my other comment.

Creating a new StatusCode, called `StatuseCode.UNINITIALIZED` could also be a solution, if creating a new statuscode is allowed. In my mind, when something gets disposed, it gets uninitialized, thus having a statuscode that indicates this directly, instead of having to decide between two, might be an alternative solution.

Reply:
I refactored `dispose`, but the flag still is on the job's constructor because it does seem like an overkill to make two job classes, each calling a different method, so perhaps changing the parameter to a StatuCode would make more sense, but that also seems to be violating some rules too.
I kind of like the idea of a new status code, but I feel like the semantics and reporting logic of it require a lot of thought.
For a regular user looking at the Rundown View, it should be clear whether performing any playout action might fail due to any reason - could be that the Playout gateway has/was restarted, could be that some device is misconfigured etc.. From that perspective these are all bad states and for example if the user does a Take, it will probably look bad on the output. On the other hand, the detailed cause of that state, relevant for a superuser or support, is in the message, but I can see that for the purpose of health monitoring, a distinction between an expected initialization (e.g. due to settings change or a restart triggered by the user) could be useful to assess the severity and need for intervention.